tests/unit/app/core/test_pipeline.py

```diff
@@ -1,20 +1,26 @@
+import json
 from typing import Dict
+from unittest.mock import patch
 
+from click.testing import CliRunner
 from sqlalchemy.orm import Session
 
-from app.core.ingestion.pipeline import (
-    flatten_pipeline_metadata,
+from app.repository.pipeline import (
+    _flatten_pipeline_metadata,
     generate_pipeline_ingest_input,
 )
+from app.service.pipeline import format_pipeline_ingest_input, get_db_state_content
+from scripts.db_state_validator_click import main as db_state_validator_main
 from tests.non_search.setup_helpers import (
     setup_docs_with_two_orgs,
     setup_with_documents_large_with_families,
+    setup_with_two_docs_multiple_languages,
     setup_with_two_docs_one_family,
     setup_with_two_unpublished_docs,
 )
 
 
-def test_generate_pipeline_ingest_input(data_db: Session):
+def test_generate_pipeline_ingest_input(data_db: Session, data_client):
```

Comment:
I think adding this results in two session objects as data_client depends on data_db, and I'd expect some clashes

Reply:
This was never meant to be pushed but slipped in. You can have both without issue though I believe - we use this in other tests :)

Reply:
Ah fair enough

Reply:
Have removed :)